packages/muelu/test/scaling/DatabaseSchwarzSmoo.xml

```diff
@@ -0,0 +1,52 @@
+<!-- Running
+         mpirun -np 4 MueLu_Driver.exe __linAlgebra=Tpetra __xml=blkSmooEquivOlapSchwarz.xml __belosType=\"Fixed\ Point\" __rowmap=GblkMap.dat __matrix=GblkAmat.dat __rhs=GblkRhs.dat __tol=.1 __userBlks=regularOverLap.dat
+ and     mpirun -np 4 MueLu_Driver.exe __linAlgebra=Tpetra __xml=oLapSchwarzEquivBlkSmoo.xml __belosType=\"Fixed\ Point\" __rowmap=GblkMap.dat __matrix=GblkAmat.dat __rhs=GblkRhs.dat __tol=.1
+
+    should give identical convergence histories as both do overlapping block Jacobi with the same blocks. Here, __ should be replaced by double-hyphen (double-hyphen not allowed in xml comments).
+-->
+<ParameterList name="MueLu">
+  <Parameter        name="verbosity"                            type="string"   value="high"/>
+
+  <Parameter        name="number of equations"                  type="int"      value="1"/>
+
+  <Parameter        name="max levels"                           type="int"      value="1"/>
+  <Parameter        name="coarse: max size"                     type="int"      value="1"/>
+
+  <Parameter        name="multigrid algorithm"                  type="string"   value="sa"/>
+
+  <!-- ===========  AGGREGATION  =========== -->
+  <Parameter        name="aggregation: type"                    type="string"   value="uncoupled"/>
+  <Parameter        name="aggregation: drop scheme"             type="string"   value="distance laplacian"/>
+
+  <!-- ===========  SMOOTHING  =========== -->
```

Comment:
Why is this used as a coarse solver instead of a smoother?

Reply:
I didn't actually want to run multigrid because a lot of the problems I tested were high-order, and my understanding was if I wanted a smoother-only test then I needed `"coarse: type"` with a smoother in it, similar to `oLabSchwarzEquivBlkSmoo.xml` and `generalBlkSmoothing.xml`. With that said, the repartitioning options below can definitely be removed.

Reply:
The smoother-only behavior can be tested in Ifpack2. Behavior in a multigrid solver can be tested in MueLu.

Reply:
Do we have drivers in Ifpack2 for loading matrix-market data?

Reply:
Yes